server/src/client_api.rs

```diff
@@ -272,6 +275,26 @@ pub async fn post_bulk_metrics(
     );
     Ok(HttpResponse::Accepted().finish())
 }
+
+#[utoipa::path(context_path = "/api/client", responses((status = 202, description = "Accepted Instance data"), (status = 403, description = "Was not allowed to post instance data")), request_body = EdgeInstanceData, security(
+("Authorization" = [])
+)
+)]
+#[post("/metrics/edge")]
+#[instrument(skip(_edge_token, instance_data, connected_instances))]
+pub async fn post_edge_instance_data(
+    _edge_token: EdgeToken,
+    instance_data: Json<EdgeInstanceData>,
+    connected_instances: Data<RwLock<Vec<EdgeInstanceData>>>,
+) -> EdgeResult<HttpResponse> {
+    info!("Accepted {instance_data:?}");
```

Comment:
Could we check if we have an InstanceDataSender here and drop this if we don't have one? It'd reduce memory and let us unnest some code later on